biome.json

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+  "vcs": {
+    "clientKind": "git",
+    "enabled": true,
+    "useIgnoreFile": true
+  },
+  "files": {
+    "ignoreUnknown": true
+  },
+  "formatter": {
+    "enabled": true,
+    "formatWithErrors": false,
+    "indentStyle": "space",
+    "indentWidth": 2,
+    "lineEnding": "lf",
+    "lineWidth": 80,
```

Comment:
biome doesn't have support for `.editorconfig` yet so these values can be added here like lineEnding and others, however this feature is coming soon. Here's [the issue](https://github.com/biomejs/biome/issues/1724).

Keep in mind that biome uses a lot of feature flag so most of the issues that were recently merged are hidden or not even released. so they will come slowly